cassandra/src/test/java/org/apache/calcite/test/CassandraAdapterTest.java

```diff
@@ -154,4 +154,16 @@ static void load(CqlSession session) {
         .enableMaterializations(true)
         .explainContains("CassandraTableScan(table=[[twissandra, Tweets_By_User]])");
   }
+
+  @Test void testNotEquals() {
+    CalciteAssert.that()
+        .with(TWISSANDRA)
+        .query("select * from \"tweets\" where \"username\" <> 'JmuhsAaMdw'")
+        .enableMaterializations(false)
+        .explainContains("CassandraFilter(condition=[<>($0, 'JmuhsAaMdw')])\n"
+            + "    CassandraTableScan(table=[[twissandra, tweets]])")
+        .limit(1)
+        .returns("tweet_id=f3c329de-d05b-11e5-b58b-90e2ba530b12; "
```

Comment:
Is this result reasonable? The result has `username = JmuhsAaMdw`

Reply:
Sorry, this test was automatically generated by copilot before. I am setting up a Cassandra environment to test it.